model/functions.php

```diff
@@ -660,4 +660,280 @@ function getAllColumbariumRelatedDataWithFilter(ColumbariumFilter $filter)
         $response -> addError($errorMessage);
     }
     return $response;
+}
+
+function getAllContacts(){
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db -> get_connection();
+        $query = "select * from contacts;";
+        $statement = $con->prepare($query);       
+        $success = $statement->execute();
+        $result = $statement->fetchAll();
+        $statement->closeCursor();
+        if($success && count($result) > 0)
+        {
+            for($i = 0; $i<count($result); $i++)
+            {
+                $row = $result[$i];
+                $contact = new Contact($row);
+                $response ->addResult($contact);
+            }
+        }
+        if(!$success)
+        {
+            $response -> addError("Failed to fetch Contacts");
+        }
+    }    
+    catch (PDOException $e) {
+        $errorMessage = $e->getMessage();
+        $response -> addError($errorMessage);
+    }
+    return $response;
+}
+
+
+function insertBuriedIndividual(ToBuriedIndividualTable $buriedIndividual)
+{
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db -> get_connection();
+        $query = "INSERT INTO buried_individuals (FIRST_NAME, MIDDLE_NAME, LAST_NAME, MAIDEN_NAME, DOB, DOD, VETERAN, OBITUARY, TOMB_ID, COLUMBARIUM_ID) 
+            VALUES (:firstName, :middleName, :lastName, :maidenName, :dob, :dod, :veteran, :obituary, :tombId, :columbariumId);";
+        $statement = $con->prepare($query);
+        $statement->bindValue(':firstName', $firstName = $buriedIndividual->firstName);
+        if(!isset($buriedIndividual ->middleName))
+        {
+            $statement->bindValue(':middleName', null, PDO::PARAM_NULL);
+        }
+        else{
+            $statement->bindValue(':middleName', $middleName = $buriedIndividual->middleName);
+        }
+        $statement->bindValue(':lastName', $lastName = $buriedIndividual->lastName);
+        if(!isset($buriedIndividual ->maidenName))
+        {
+            $statement->bindValue(':maidenName', null, PDO::PARAM_NULL);
+        }
+        else{
+            $statement->bindValue(':maidenName', $maidenName = $buriedIndividual->maidenName);
+        }
+        $statement->bindValue(':dob', $dob = $buriedIndividual->dob);
+        $statement->bindValue(':dod', $dod = $buriedIndividual->dod);
+        $statement->bindValue(':veteran', $veteran = $buriedIndividual->veteran? 1 : 0);
```

Comment:
remove variable assignment in the second parameter as it is unnecessary. Also, remove the check for 1 and 0. This should be done in the mapping class.